build.gradle

```diff
@@ -2,20 +2,21 @@
 buildscript {
     ext {
         gradleVersion = '3.1.1'
-        kotlinVersion = '1.2.31'
```

Comment:
The latest version of kotlin is `1.3.11`. Can you put it instead?

Reply:
Fixed in commit b9f9c60